### ISSUE TITLE
$root.currentUser isn't reactive

### ISSUE THREAD

**Issue by kusmierz** [meteor/simple-todos-angular#16]
When I'm trying to login, there is no form to add new task. It will appear after page refresh. It seems it is not reactive. But if I made small change from `$root.currentUser` to `$auth.currentUser` and from `$root.currentUser._id` to `$auth.currentUser._id` in the `simple-todos-angular.html` file it works as it should.

**Comment by felicedeNigris** [comment]
I'm experiencing the same thing. I have some items in a menu that I want to hide with `(ng-show="$root.currentUser")` , but I notice it works only after I refresh the browser?

**Comment by Urigo** [comment]
Fixed with the new version of `angular-meteor-auth`
